cosmwasm/go.mod

```diff
@@ -0,0 +1,79 @@
+module cosmwasm-plugin
```

Comment:
```suggestion
module github.com/ignite/plugins/cosmwasm
```

Reply:
Resloved: [set module name to github.com/ignite/plugins/cosmwasm , fix in README](https://github.com/yerasyla/plugins/commit/440595f18652e29aef73e9783c29dc2df8a647a7)